packages/synchro-charts/src/components/sc-dial/sc-dial-base/sc-dial-svg.tsx

```diff
@@ -0,0 +1,191 @@
+import { Component, h, Prop, State, Watch } from '@stencil/core';
+import { NO_VALUE_PRESENT } from '../../../constants';
+import { DataPoint, DataStream } from '../../../utils/dataTypes';
+import { round } from '../../../utils/number';
+import { getIcons } from '../../charts/common/annotations/iconUtils';
+import { Threshold } from '../../charts/common/types';
+import { DialSizeConfig } from '../type';
+import { ColorConfigurations, getFontWidth } from '../util';
+
+const FONT_SIZE = 48;
+const ICON_SIZE = 24;
+const LABEL_SIZE = 24;
+const UNIT_SIZE = 24;
+const DIAL_THICKNESS = 34;
+const R = 138;
```

Comment:
R?

Reply:
By the UX, the circle's diameter is 138. So `R` is diameter.

Reply:
R to represent diameter is confusing. maybe call is DIAMETER?

Reply:
Done --- Change `R` to `DIAMETER`.